### PR TITLE
SYNR-1279

### PR DESCRIPTION
**Pull request by kimyen**

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+## Available Utilities
+
+### Copy
+
+#### Copy Entity (File, Folder, Table, Link, and Project)
+
+The example below copy File ID `syn123` to Project ID `syn456`:
+```{r eval=F}
+  copy("syn123", "syn456")
```
Comment (karawoo) [inline README.Rmd]: It looks like there's some extra space here and in a couple other code chunks.

Comment (karawoo) [inline README.Rmd]: It looks like there's some extra space here and in a couple other code chunks.

File: README.Rmd [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+```
+
+The first parameter in `copy()` can be a File ID, a Table ID, or a Link ID. The second parameter must be a Folder ID or a Project ID.
```
Comment (karawoo) [inline README.Rmd]: It appears that copying a folder also works. The documentation for copy() doesn't state what types of entities can be copied; can we put this information there as well?

File: README.Rmd (hunk 1 above)
Comment (kimyen) [reply README.Rmd]: Thanks @karawoo! Do you refer to the spaces before `copy` or the spaces between the parameters?

Comment (karawoo) [reply README.Rmd]: The space before `copy`. I like the space between the parameters.

File: README.Rmd (hunk 2 above)
Comment (kimyen) [reply README.Rmd]: For this topic, the `copy` function in the Python client is not tested. I want to hold off on advertising copying folder (and possibly recursively copy the entire hierarchy) until we refactor and have all test coverage for this function. This refactor would also include the re-write/ edit the `copy` function documentation in Python. synapserutils will then inherit the new docs. I will update this part then.

File: README.Rmd [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+synLogin()
+```
+
+## Available Utilities
+
+### Copy
+
+#### Copy Entity (File, Folder, Table, Link, and Project)
+
+The example below copy File ID `syn123` to Project ID `syn456`:
```
Comment (brucehoff) [inline README.Rmd]: copies

File: README.Rmd [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "README-"
+)
+```
+
+# synapserutils
+
+The `synapserutils` package provides a set of utility functions, built on top of [`synapser`](http://sage-bionetworks.github.io/synapser/) package.
```
Comment (brucehoff) [inline README.Rmd]: built on top of the synapser package

File: README.Rmd [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+The first parameter in `copy()` can be a File ID, a Table ID, or a Link ID. The second parameter must be a Folder ID or a Project ID.
+
+For more information on `copy()`, please see:
+```{r eval=F}
+?copy
+```
+
+#### Copy Wiki and Wiki Subpage
+
+To copy a wiki page that associate with an Entity (Project/ File/ Folder/ Table) `syn123` to another Entity `syn789`:
```
Comment (brucehoff) [inline README.Rmd]: that is associated with

File: README.Rmd [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+### Batch Process
+
+#### Upload Data in Bulk
+
+`syncToSynapse()` takes a manifest file and uploads the files listed in the manifest file to Synapse:
+```{r eval=F}
+syncToSynapse("/path/to/manifest.tsv")
+```
+
+The manifest file format and instructions on how to create one could be found [here](http://docs.synapse.org/articles/uploading_in_bulk.html#creating-a-manifest) and by:
```
Comment (brucehoff) [inline README.Rmd]: can be found here

**Comment by brucehoff** [comment]
Looks good!

**Pull request merged**